readme.md

```diff
@@ -22,6 +24,8 @@ It's simple, all you need to do is iterate through the array checking if its ele
 
 ### Pseudocode implementation
 
+Have in mind that this pseudocode is just an example, we strongly encourage you to code a different approach. 
```

Comment:
@gustavo-depaula take a look ath it, I guess that it'll help to people not code only like this pseudo

Reply:
idk... your choice!